tests/api/test_ingress.py

```diff
@@ -20,7 +20,7 @@ async def test_validate_session(api_client, coresys):
         "aiohttp.web_request.BaseRequest.__getitem__",
         return_value=coresys.homeassistant,
     ):
-        resp = await api_client.post("/ingress/session")
+        resp = await api_client.post("/ingress/session", json={"user-id": "some-id"})
```

Comment:
please make also a new test without user_id for make sure old core version still works for now